File: 8kyu/convert-number-to-reversed-array-of-digits/convert-number-to-reversed-array-of-digits.go
package main

import (
	"fmt"
	"strconv"
)

func Digitize(n int) []int {
  nStr := strconv.Itoa(n)
	digits := make([]int, 0)

	for i := len(nStr) - 1; i >= 0; i -= 1 {
		digit, _ := strconv.Atoi(string(nStr[i]))
		digits = append(digits, digit)
	}
  return digits
}


func main() {
	fmt.Println(Digitize(35231))
}

File: 8kyu/convert-number-to-reversed-array-of-digits/convert-number-to-reversed-array-of-digits.py
# def digitize(n):
#   n, digits = str(n), []
#   for num in n:
#     digits.append(int(num))
#   return digits[::-1]
  
def digitize(n):
  return list(map(int, str(n)))[::-1]

print(digitize(35251))